src/index.html

```diff
@@ -2,31 +2,64 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Document</title>
+
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <main class="main">
+      <div class="main__form">
+        <form
+          class="search-form search-form--large"
+          data-qa="big"
+        >
+          <span
+            href="#"
+            class="search-form__search-icon search-form__search-icon--large"
+          ></span>
```

Comment:
The <span> element should not have an 'href' attribute, as it is not a link but a span element. If you need to create a clickable icon, consider using the <a> or <button> element instead.